lib/widgets/padding.dart

```diff
@@ -8,7 +16,16 @@ import '/args/edge_insets.dart';
 extension PaddingConverter on Padding {
   /// Converts the [Padding] to a [pw.Padding].
   pw.Padding toPdfWidget(pw.Widget? child) => pw.Padding(
-        padding: (padding as EdgeInsets).toPdfEdgeInsets(),
+        padding: _convertEdge(padding),
```

Comment:
```suggestion
        padding: padding.toPdfEdgeInsets(),
```

Use the `EdgeInsetsGeometryConverter`.